test/recipes/99-test_fuzz_x509.t

```diff
@@ -15,6 +15,9 @@ use OpenSSL::Test::Utils;
 my $fuzzer = "x509";
 setup("test_fuzz_${fuzzer}");
 
+plan skip_all => "This test requires $fuzzer support"
+    if disabled("ocsp");
```

Comment:
is there no way how the test could be compiled with #ifndef OPENSSL_NO_OCSP?

Reply:
In theory, yes. Since it's the last part of the file, it will currently then ignore part of the file it reads, and just not cover as much. If at some later point, the test is extended to read more data, the files will not have proper coverage.